sql/vector_range_collection.go

```diff
@@ -0,0 +1,23 @@
+package sql
+
+import "fmt"
+
+type OrderAndLimit struct {
```

Comment:
file name and contents seem a bit at odds. and you'll get license formatting complaints

Reply:
Added this class to `sql/core.go`